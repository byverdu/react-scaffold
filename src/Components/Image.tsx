import React from 'react';
import logo from 'logo.svg';

const Image: React.FC = () => <img src={logo} className="App-logo" alt="logo" />;

export default Image;
