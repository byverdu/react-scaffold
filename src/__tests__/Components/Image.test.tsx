import React from 'react';
import Image from 'Components/Image';
import enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('<Image />', () => {
  const wrapper: enzyme.ShallowWrapper = shallow(<Image />);

  it('should create a snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
