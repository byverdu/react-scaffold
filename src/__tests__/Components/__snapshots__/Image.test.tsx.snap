// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> should create a snapshot 1`] = `
<img
  alt="logo"
  className="App-logo"
  src="logo.svg"
/>
`;
